R/run_trial.R

```diff
@@ -149,6 +150,49 @@ run_trial <- function(
       )
     }
 
+    # post-processing to get evaluation metrics
+    if(design$evaluation$type %in% target_types_auc) {
+      auc_eval <- calc_auc_from_regimen(
+        regimen = res$final_regimen,
+        parameters = pars_true_i, # true patient parameters
+        model = design$sim$model,
+        target_design = design$evaluation,
+        covariates = covs
+      )
+      eval_time <- get_sampling_times_from_scheme(
+        design$evaluation$scheme,
+        res$final_regimen
+      )
+      eval_exposure <- rbind(
+        eval_exposure,
+        data.frame(
+          id = i,
+          time = eval_time,
+          auc = auc_eval
+        )
+      )
+    } else if (design$evaluation$type %in% target_types_conc){
```

Comment:
if we want to allow for both trough and auc outputs, we could change this to an `if` instead of an `else if`